zerver/lib/exceptions.py

```diff
@@ -715,3 +716,19 @@ def __init__(self, first_parameter: str, second_parameter: str) -> None:
     @override
     def msg_format() -> str:
         return _("Incompatible values for '{first_parameter}' and '{second_parameter}'.")
+
+
+class CannotDeactivateGroupInUseError(JsonableError):
+    code = ErrorCode.CANNOT_DEACTIVATE_GROUP_IN_USE
```

Comment:
Using a different code, so that client can show error accordingly.

Reply:
Yes, sounds good.